src/index.js

```diff
@@ -2,17 +2,27 @@ import { createFilter } from 'rollup-pluginutils';
 import postcss from 'postcss';
 import styleInject from 'style-inject';
 import path from 'path';
+import fs from 'fs';
 
 function cwd(file) {
   return path.join(process.cwd(), file);
 }
 
+function extract(mappings, destination) {
+  const output = Object.keys(mappings)
+    .map(file => mappings[file])
+    .join('');
+  fs.writeFileSync(destination, output);
+}
+
```

Comment:
This seems to limit the extraction to a single file.

This is acceptable for a website/webapp that loads all its CSS at once.
But, less ideal for a website/webapp that loads stylesheets as needed.

Can this feature addition also allow extracting _N_ stylesheets from _N_ stylesheets?